src/serac/physics/base_physics.cpp

```diff
@@ -108,7 +91,7 @@ void BasePhysics::setShapeDisplacement(FiniteElementState& shape_displacement)
   shape_displacement_ = shape_displacement;
 }
 
-void BasePhysics::outputState(std::optional<std::string> paraview_output_dir) const
+void BasePhysics::outputStateToDisk(std::optional<std::string> paraview_output_dir) const
```

Comment:
I agree!  And the name here outputStateToDisk seems to be the 'best' usage of the term 'state'.

Reply:
maybe InternalVariables (I'm trying to see if we have a capitalization, _ naming style set).  Also, InternalVariableArray may be better.  Class names should basically never be plural.  Because then people make vector<InternalVariables> and have to say 'internal variables'-es.  I got in trouble with this naming some years ago writing a simd library, and everyone had a good laugh at me for a few years cause it was impossible to talk about in conversations.

Reply:
Sorry, last suggestion, I promise:

InternalVariableField

Reply:
I'll put up an issue on this because I think these objects now have enough usage to come up with better names.